json_server/main.py

```diff
@@ -14,8 +13,13 @@ def cli_command(filename, port):
 
     @app.route("/", methods=["GET"])
     def index():
-        with open(click.format_filename(filename)) as file:
-            contents = json.loads(file.read())
-        return jsonify(contents)
+        with open(click.format_filename(filename)) as f:
+            try:
+                contents = json.loads(f.read())
+            except json.JSONDecodeError as e:
+                return f"Invalid JSON\nError - {e}"
```

Comment:
`raise` an error instead of just returning a formatted string.
Feel free to write a custom `Exception` class if you want.

Reply:
Ok I will make a custom exception

Reply:
Ok sure